### DISCUSSION TITLE
Show Armor and Off Hand slots in InvSee

### DISCUSSION THREAD

**Discussion by Gabo6480** [EssentialsX/Essentials#5135] [Ideas and Feature Suggestions]
InvSee curently only shows the main inventory of the player, IE the hotbar and the 3x9, which means that players can equip items in the offhand or armor slots and admins won't be able to do nothing about it.

**Comment by Evidentsinger14** [comment]
You can view armor slots, and offhand by putting *anything* after the invsee command, but If i remember correctly, you're unable to actually modify those slots. As per [this](https://github.com/EssentialsX/Essentials/discussions/4624) thread, allowing for modification of those is out of scope of EssentialsX.

**Comment by Chew** [comment]
Should be covered in https://github.com/EssentialsX/Essentials/issues/3956#issuecomment-771694504
